prody/proteins/blastpdb.py

```diff
@@ -235,15 +240,15 @@ def getParameters(self):
 
         return dict(self._param)
 
-    def getHits(self, percent_identity=90., percent_overlap=70., chain=False):
+    def getHits(self, percent_identity=0., percent_overlap=0., chain=False):
```

Comment:
It is better not to change these default values for compatible reasons. Changing default values of a long-standing function could be a bad practice as it may change the results of the same code and it may frustrate the users. A most conserved point of view is that for the sake of compatibility, unless necessary, low-level and old functions should not be modified if the output may change. Let me know if the original values are not good.

Reply:
It's not the expected behaviour for new users. I was expecting that getHits would get all the hits unless I specified filters and I would think other people would too.

I've changed it back anyway.